nodewatcher/modules/monitor/http/survey/processors.py

```diff
@@ -111,17 +109,24 @@ def get_node_snr(self, context, node, frequency_band='2'):
         :param frequency_band: either '2' or '5', corresponding to 2.4GHz and 5GHz.
         :return: SNR of the node at the specified frequency band.
         """
-
+        edges = {}
+        vertices = {}
         try:
             for interface in context.http.core.wireless.interfaces:
-                if str(context.http.core.wireless.interfaces[interface].frequency)[:1] == frequency_band:
-                    signal = context.http.core.wireless.interfaces[interface].signal
-                    noise = context.http.core.wireless.interfaces[interface].noise
-                    if (not signal or not noise):
-                        self.logger.warning("No SNR available for " + str(node))
-                        return 0
-                    else:
-                        return signal - noise
+                # create an edge for every interface
+                edge = {}
+                edge = {'f': str(node), 't': destination_id}
```

Comment:
I'm currently using the name of the node for the 'from' field but I am using BSSIDs for the 'to' field. What do you think? Should I use BSSIDs or MAC addresses or something else for the source node? The benefit of the current implementation is that we only have one source node, whereas BSSIDs differ for different interfaces.